butter-cms.json

```diff
@@ -68,14 +68,14 @@
     "I18N Localized Content": "Yes",
     "Content Trees": "No",
     "Tagging": "Yes",
-    "Content Relations": "",
-    "Nesting of Fields": "No",
+    "Content Relations": "Yes",
+    "Nesting of Fields": "Yes",
     "GDPR Statement": "No",
     "GDPR API": "No",
     "Editing Conflict Detection": "No",
     "Versioning": "No",
     "Content Scheduling": "No",
-    "Content Branches": "No",
+    "Content Branches": "Yes",
```

Comment:
https://buttercms.com/blog/manage-multiple-environments-and-client-websites-our-multisite-dashboard